velox/core/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ add_library(velox_core Expressions.cpp PlanFragment.cpp PlanNode.cpp
 target_link_libraries(
   velox_core
   velox_config
+  velox_exec
```

Comment:
velox_core should not depend on velox_exec as this introduces circular dependency